bin/link_check.py

```diff
@@ -39,14 +43,15 @@ def test_ftp(url):
         req = urllib.request.Request(url)
         if proxy:
             req.set_proxy(proxy, 'http')
-        response = urlopen(req, timeout=30)
+        response = urlopen(req, timeout=URL_TIMEOUT)
         chunk = response.read(16)
         if len(chunk) == 16:
             res.status_code = 200
         else:
             res.status_code = 404
     except:
-        res.status_code = 404
+        print('ftp exception', url)
+        return Exception()
```

Comment:
bare except clauses are a bad idea: they will catch much more than you expect (like pressing CTRL+C while the script is running). returning an empty exception object is also very strange. is there another way you can think of doing this?